sdk/python/feast/job_service.py

```diff
@@ -55,6 +55,10 @@
 def _job_to_proto(spark_job: SparkJob) -> JobProto:
     job = JobProto()
     job.id = spark_job.get_id()
+    log_uri = spark_job.get_log_uri()
```

Comment:
looks strange.
I suggest one-liner
`log_uri = spark_job.get_log_uri() or ""`
or
`log_uri = bool(spark_job.get_log_uri()) and spark_job.get_log_uri(`

Reply:
I chose to do the current approach rather than the one liner approach because the latter failed the python linting

Reply:
Specifically, get_log_uri returns Optional str, hence the linter consider it as type mismatch, even though the condition ensure that it will never be None